src/components/project_admin/suite_settings.vue

```diff
@@ -50,19 +48,50 @@
           </template>
         </select-object>
       </div>
-
       <div class="toggle-container form-field-wrapper">
-        <toggle v-model="d_suite.allow_network_access"
-                @input="$emit('field_change', d_suite)"
-                ref="allow_network_access">
+        <div>
+          <i class="fas fa-exclamation-triangle warning-tip"></i>
+          <tooltip width="large" placement="right">
```

Comment:
You can make it so that the warning triangle is the thing to hover over to show the tooltip using the "trigger" slot. Off the top of my head, that would look something like:

```
<tooltip width="large" placement="right">
  <template v-slot:trigger>
    <i class="fas fa-exclamation-triangle warning-tip"></i>
  </template>
  {{WARNING_MESSAGE}}
</tooltip>
```